hack/modules/install_all_apps.sh

```
@@ -0,0 +1,66 @@
+    return 1
+}
+
+function install_all_apps() {
+    local charts_dir="$1"
+    local namespace="$2"
+    local repo_name="$3"
+    local repo_ns="$4"
+
+    local ignore_file="./modules/ignored_charts"
+    local ignored_charts
+    ignored_charts=($(load_ignored_charts "$ignore_file"))
```

Shellcheck

[warning] Prefer mapfile or read -a to split command output (or quote to avoid splitting).

(SC2207)